homeassistant/components/smartthings/sensor.py

```diff
@@ -116,7 +117,7 @@
         )
     ],
     Capability.illuminance_measurement: [
-        Map(Attribute.illuminance, "Illuminance", "lux", DEVICE_CLASS_ILLUMINANCE)
+        Map(Attribute.illuminance, "Illuminance", LIGHT_LUX, DEVICE_CLASS_ILLUMINANCE)
```

Comment:
This is a breaking change; unit of measurement changed from `lux` to `lx`, which can affect things like InfluxDB.